routes/room.go

```diff
@@ -309,3 +309,89 @@ func DisconnectUserFromRoom(c *gin.Context) {
 
 	c.JSON(http.StatusNoContent, nil)
 }
+
+// KickUserFromRoom godoc
+// @Summary      Kicks a user from a room.
+// @Tags         Rooms
+// @Security     BasicAuth
+// @Produce      json
+// @Param        id path int true "Room ID"
+// @Param        userid path int true "User ID"
```

Comment:
Indent bro wtf

Reply:
Just checking if you learned from your mistakes !